.github/workflows/release_pull_request.yml

```diff
@@ -18,7 +18,22 @@ jobs:
       - run: go build github.com/$GITHUB_REPOSITORY
       - uses: github/codeql-action/analyze@v2
 
+  generate-ui:
+    if: startsWith(github.head_ref, 'release-please-') == true
```

Comment:
It would be great to build the UI on every PR. If there are problems with the build, I think we'd like to learn about them as soon as possible.

Could we have this run without the `if` ?

Reply:
This definitely needs to be generated for all pull requests. I initially thought the UI is only necessary for the image but your points in #1882 and the failing tests prove that to not be true. This should be changed to a prereq for test jobs in both CI and local dev i.e. the Makefile

Reply:
We should not require someone to build the UI to be able to build the Go binary, or run the Go tests.

I think we can make that work, but generating the UI in every PR is nice anyway so that we verify the build still works in advance of the release.